CMakeLists.txt

```diff
@@ -9,6 +9,42 @@ project(luthier
 set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
+include(FetchContent)
+
+#Conditional to check if LLVM_SRC_DIR exists somewhere or not
+#Check if LUTHIER_LLVM_SRC_DIR is empty string or just a folder
+if (NOT IS_DIRECTORY LUTHIER_LLVM_SRC_DIR AND CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+
+  #Fetches GIT_HASH variable
+  execute_process(
```

Comment:
Use the `file` command with `REGEX` here, there shouldn't be a need for a Python script for this:
https://cmake.org/cmake/help/latest/command/file.html#reading